internal/confmapprovider/discovery/bundle/bundle.d/receivers/oracledb.discovery.yaml.tmpl

```
@@ -0,0 +1,50 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+oracledb:
```

to tie into basic component validation:
```suggestion
{{ receiver "oracledb"}}:
```